sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -55,6 +66,7 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
         });
 
         this.itemViews = itemCards;
+        this.updateReadStatus();
```

Comment:
Let's say you read every item in every feed. Each category will still be marked as "New" in the sidebar since we're not tracking read/unread by counting items anymore. So, this makes sure that when you click on a Category that says "New" with all read items in it, it updates the read status and marks it as read. 

We need this because, otherwise, there is no change on "unread" so the category will continue to be marked as unread even if every item is read since there's no change.